scripts/tools/generate_esp32_chip_factory_bin.py

```diff
@@ -28,6 +29,10 @@
 from bitarray import bitarray
 from bitarray.util import ba2int
 
+CHIP_TOPDIR = os.path.dirname(os.path.realpath(__file__))[:-len(os.path.join('scripts', 'tools'))]
+sys.path.insert(0, os.path.join(CHIP_TOPDIR, 'scripts', 'tools', 'spake2p'))
+from spake2p import generate_verifier  # noqa: E402
```

Comment:
pep8 will put this up top and script won't find the spake2p module, so skipping that check here